### PR TITLE
Update plugin to the v2.0 of the api

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,7 @@
 source 'https://rubygems.org'
-gemspec
\ No newline at end of file
+gemspec
+gem "logstash-core", :path => "./logstash/logstash-core"
```
Comment (andrewvc) [inline Gemfile]: You probably didn't want these in the PR

Comment (ph) [inline Gemfile]: I do

Comment (ph) [inline Gemfile]: for more context see this https://github.com/elastic/logstash/issues/5227

**Comment by suyograo** [comment]
LGTM

**Pull request merged**